conf.d/cassandra.yaml.example

```diff
@@ -40,7 +40,9 @@ init_config:
           - MemtableSwitchCount
           - MinRowSize
           - PendingTasks
+          - RangeOpertaions
```

Comment:
Looks like a typo.
Also RangeOperation seems to be in the org.apache.cassandra.service domain, not org.apache.cassandra.db.